### DISCUSSION TITLE
[Q/A][Java SDK (Component)] xxx[DUBBO] Model start failed: Dubbo Module[1.1.1] start failed: java.lang.IllegalStateException: Extension instance (name: tri, class: interface org.apache.dubbo.rpc.Protocol) couldn't be instantiated: null, dubbo version: 3.2.13, current host: 10.32.9.191, error code: 5-14. This may be caused by , go to https://dubbo.apache.org/faq/5/14 to find instructions. java.lang.IllegalStateException: Extension instance (name: tri, class: interface org.apache.dubbo.rpc.Protocol) couldn't be instantiated: null

### DISCUSSION THREAD

**Discussion by louqing13129** [apache/dubbo#14893] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

3.2.13 version 
[DUBBO] Model start failed: Dubbo Module[1.1.1] start failed: java.lang.IllegalStateException: Extension instance (name: tri, class: interface org.apache.dubbo.rpc.Protocol) couldn't be instantiated: null, dubbo version: 3.2.13, current host: 10.32.9.191, error code: 5-14. This may be caused by , go to https://dubbo.apache.org/faq/5/14 to find instructions. java.lang.IllegalStateException: Extension instance (name: tri, class: interface org.apache.dubbo.rpc.Protocol) couldn't be instantiated: null
yml 
dubbo:

  protocol:
    name: tri
    port: -1

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please provide the full stack
